bedrock/mozorg/templates/mozorg/plugincheck.html

```diff
@@ -239,11 +238,21 @@ <h1 class="large">{{_('Check Your Plugins')}}</h1>
     </div>
     <p><a href="#top">{{_('Back to Top')}}</a></p>
 </section>
+
+<!-- dislayed to nonFx users -->
+<div class="not-supported">
+  <header id="main-feature">
+      <h1 class="large">{{_('Check Your Plugins')}}</h1>
+      <p class="preamble">{{_('Keeping your plugins up to date helps Firefox run safely and smoothly.')}}</p>
+  </header>
+
+  <section>
+    <p>{{_('Currently the plugin check service is only available to Firefox users.')}}</p>
```

Comment:
@flodolo nope, no other string changes. So, just no `else` for the l10n_has_tag? It did not exist before so, there is no else ;)